### PR TITLE
fix for file job creation on gengo

### PR DESCRIPTION
**Pull request by deepak-adobe**

### CODE REVIEW

**Comment by hayate** [comment]
LGTM

**Comment by hayate** [comment]
@seikoudoku2000 can you review this PR too please?

File: src/main/java/com/gengo/client/GengoClient.java [hunk 1]
```diff
@@ -140,6 +140,34 @@ public JSONObject postTranslationJobs(List<TranslationJob> jobs, boolean process
...
+    	    {
+    	        try
+    	        {
+    	            String url = baseUrl + "translate/jobs";
+    	            JSONObject data = new JSONObject();
+    	            /* We can safely cast our list of jobs into a list of the payload base type */
+    	            @SuppressWarnings({ "rawtypes", "unchecked" })
+    	            List<Payload> p = (List)jobs;
+    	            data.put("jobs", (new Payloads(p)).toJSONArray());
+    	            data.put("as_group", processAsGroup ? MYGENGO_TRUE : MYGENGO_FALSE);
```
Comment (seikoudoku2000) [inline src/main/java/com/gengo/client/GengoClient.java]: I think file job can't be a group job.   @hayate ?

Comment (hayate) [inline src/main/java/com/gengo/client/GengoClient.java]: good catch, thanks @seikoudoku2000 

@deepak-adobe could you remove the `"as_group"` line?

Comment (deepak-adobe) [inline src/main/java/com/gengo/client/GengoClient.java]: I figured the file jobs for same locale can be grouped together. won't it?

Comment (hayate) [inline src/main/java/com/gengo/client/GengoClient.java]: no it won't, as the back-end does not support grouping files jobs.

**Comment by deepak-adobe** [comment]
removed the group param for file jobs.

**Comment by seikoudoku2000** [comment]
LGTM :sunny:

**Comment by hayate** [comment]
thank you @deepak-adobe

**Pull request merged**